docs/admin/components.mdx

```diff
@@ -426,6 +426,20 @@ All Payload fields support the ability to swap in your own React components. So,
 | **`Cell`**   | Used in the `List` view's table to represent a table-based preview of the data stored in the field. [More](#cell-component) |
 | **`Field`**  | Swap out the field itself within all `Edit` views. [More](#field-component)                                                 |
 
+There are fields that support **`Label`** and **`Error`** that are swappable likewise. The main idea of these properties is to extend existing fields without reimplementing them. The only fields that **DON'T** support are [Array](/docs/fields/array), [Blocks](/docs/fields/blocks), [Collapsible](/docs/fields/collapsible), [Group](/docs/fields/group), [Rich Text](/docs/fields/rich-text), [Row](/docs/fields/row), [Tabs](/docs/fields/tabs), and [UI](/docs/fields/ui) Fields.
```

Comment:
I would replace this paragraph with the following, but keep your nicer formatting.

> As an alternative to replacing the entire Field component, you may want to keep the majority of the default Field component and only swap components within. This allows you to replace the Error or Label within a field component or add additional components inside the field with BeforeInput or AfterInput. BeforeInput and AfterInput is allowed in any  fields that don't contain other fields or RichText.